images/component-starter-server-image/pom.xml

```diff
@@ -89,8 +89,17 @@
                   <MEECROWAVE_BASE>${docker.talend.dir.app}</MEECROWAVE_BASE>
                   <MEECROWAVE_PID>${docker.talend.dir.app}conf/server.pid</MEECROWAVE_PID>
                   <MEECROWAVE-PROPERTIES>${docker.talend.dir.app}conf/meecrowave.properties</MEECROWAVE-PROPERTIES>
-                  <CLASSPATH>${docker.talend.dir.app}lib/*:${docker.talend.dir.app}custom/*:${docker.talend.dir.base}extensions/*:${docker.talend.dir.app}resources:${docker.talend.dir.app}classes</CLASSPATH>
-                  <JDK_JAVA_OPTIONS>--add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.nio=ALL-UNNAMED --add-opens=java.base/sun.net=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED</JDK_JAVA_OPTIONS>
+                  <CLASSPATH>
+                    ${docker.talend.dir.app}lib/*:${docker.talend.dir.app}custom/*:${docker.talend.dir.base}extensions/*:${docker.talend.dir.app}resources:${docker.talend.dir.app}classes
+                  </CLASSPATH>
+                  <JDK_JAVA_OPTIONS>
+                    -Dtalend.component.starter.security.csp="default-src 'self' 'unsafe-eval'; connect-src 'self' https://unpkg.com; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline'; img-src 'self' data: blob: https://unpkg.com; font-src 'self' data: blob:; frame-ancestors 'none'"
```

Comment:
I think that it's preferable to change default property value in 
`@ConfigProperty(name = "talend.component.starter.security.csp", defaultValue = ...`   StartConfiguration class rather overloading a default (insecure value). 
On image launch, we still have the opportunity to override defaults.